File: Javascript_code/bubblesort.js
//Mohammed Azad
//This is the implementation of the bubble sort in JavaScript
function bubbleSort(array){
    var i = 0;
    var j = 0;
    var holder = 0;
    for(i=0; i < array.length; i++) { 
        for(j=0; j < array.length - i - 1; j++) { 
            if(array[j] > array[j+1]) { 
                holder = array[j];
                array[j] = array[j+1];
                array[j+1] = holder;
            }
        }
    }
}
//creating the array and then printing out the array 
var array = [5, 6, 2, 7, 1, 234, 8];
console.log("This is the array before it is sorted: ");
console.log(array);
bubbleSort(array);
console.log("This is the sorted version of the array: ");
console.log(array);

File: Python_code/bubblesort.py
#Mohammed Azad
#This is the implementation of the bubble sort using python

#making the array:
array = [1, 234, 43, 14, 1, 45, 7]

#making the function:
def bubbleSort(array):
    size = len(array)
    for i in range(size):
        for j in range(0,size-i-1):
            if array[j] > array[j+1]:
                holder = array[j]
                array[j] = array[j+1]
                array[j+1] = holder

#Running the sort
print("This is the array before it was sorted")
print(array)
bubbleSort(array)
print("This is the sorted array")
print(array)

